demisto_sdk/commands/common/tests/integration_test.py

```diff
@@ -1091,6 +1090,20 @@ def test_not_fetch(self, capsys):
         self.validator.current_file['script']['isfetch'] = False
         assert self.validator.is_valid_fetch(), 'is_valid_fetch() returns False instead True'
 
+    @pytest.mark.parametrize(argnames='marketpalces, configs, expected_is_valid', argvalues=[
+        ([MarketplaceVersions.MarketplaceV2.value], INCIDENT_FETCH_REQUIRED_PARAMS, False),
+        ([MarketplaceVersions.XSOAR.value], INCIDENT_FETCH_REQUIRED_PARAMS, True),
+        ([MarketplaceVersions.MarketplaceV2.value], ALERT_FETCH_REQUIRED_PARAMS, True),
```

Comment:
it will be easier to read
```suggestion
        ([MarketplaceVersions.MarketplaceV2.value], ALERT_FETCH_REQUIRED_PARAMS, True),
        ([MarketplaceVersions.XSOAR.value], INCIDENT_FETCH_REQUIRED_PARAMS, True),
```